### DISCUSSION TITLE
Are application/vnd.docker.distribution.manifest.v1 docker manifests supported by trivy?

### DISCUSSION THREAD

**Discussion by camaeel** [aquasecurity/trivy#6146] [Q&A] [answered]
### Question

I wanted to scan image: quay.io/uswitch/vault-creds:v0.14.0 but it seems trivy neither can pull it nor scan it. 
When I pull it on my own and run trivy scan: `docker pull quay.io/uswitch/vault-creds:v0.14.0 && trivy i quay.io/uswitch/vault-creds:v0.14.0` it reports:

```
Number of language-specific files: 0
```

Manifest has MediaType: `application/vnd.docker.distribution.manifest.v1+prettyjws` so maybe this is the source of rhe issue

### Target

Container Image

### Scanner

Vulnerability

### Output Format

None

### Mode

None

### Operating System

macos

### Version

```bash
❯ trivy --version
Version: 0.48.3
Vulnerability DB:
  Version: 2
  UpdatedAt: 2024-02-16 06:12:10.483953515 +0000 UTC
  NextUpdate: 2024-02-16 12:12:10.483953215 +0000 UTC
  DownloadedAt: 2024-02-16 07:22:51.381622 +0000 UTC
Java DB:
  Version: 1
  UpdatedAt: 2024-02-12 00:45:04.687521318 +0000 UTC
  NextUpdate: 2024-02-15 00:45:04.687521198 +0000 UTC
  DownloadedAt: 2024-02-12 09:02:10.615217 +0000 UTC
```
```

**Comment by knqyf263** [comment, marked as answer]
Not supported.
https://github.com/google/go-containerregistry/issues/377

**Comment by sarasensible** [comment]
I see that this isn't supported, but this is causing the trivy operator to go into crash loop backoff.  Is there a workaround to disable scanning images of this type?
